contrib/nydusd-http-server/README.md

```diff
@@ -0,0 +1,105 @@
+# Http server for nydusd
```

Comment:
Better to reference this doc in the section `Currently Nydus includes following tools` of main page.